README.md

```diff
@@ -153,7 +155,7 @@ For now, the logger provides two output type: `pretty` and `json`, default is `j
       
     logger.setNamespaces('namespace:*');
     logger.setLevel('debug');
-    //logger.setOutput('json');
+    //logger.setOutput(logger.output.json);
```

Comment:
I think we should add a comment on why it's commented

```javascript
// because json is the default output, it's the same as
// logger.setOutput(logger.output.json);
```